webview-ui/.eslintrc.json

```diff
@@ -1,3 +1,4 @@
 {
-	"extends": "react-app"
+	"extends": "react-app",
+	"ignorePatterns": ["!.storybook"]
```

Comment:
See: https://stackoverflow.com/questions/76572567/lint-staged-throwing-eslint-warning-for-ignored-storybook-config-file